data-service/pom.xml

```diff
@@ -55,6 +55,7 @@
         <gson.version>2.8.5</gson.version>
         <scala.version>2.13</scala.version>
         <akka.version>2.6.10</akka.version>
+        <akka.http.version>10.1.11</akka.http.version>
```

Comment:
Can we use a more current version?

Reply:
Nope, clashes with `alpakka-s3` using this version.
Maven shade is a huge pain in this case, and the downsides from the version downgrade are negligible (compare with FRS for the differences, but they are few and far between, e.g. the `newServerAt(..)` method is different here)